Makefile

```diff
@@ -13,13 +15,21 @@ PYODIDE_CXX=$(PYODIDE_ROOT)/ccache/em++
 
 CC=emcc
 CXX=em++
+
 OPTFLAGS=-O2
-CFLAGS=$(OPTFLAGS) -g -I$(PYTHONINCLUDE) -fPIC \
-	-Wno-warn-absolute-paths -Werror=int-conversion -Werror=incompatible-pointer-types \
+
```

Comment:
Not sure about the extra newlines here and above. Do they really help readability?

Reply:
yeah, it helps to better keep track when options are changed or added. I currently do some experiments with the compiler flags, and as the Makefile is maintained by hand and not generated, it makes sense to improve readability and tracking of changes whenever possible.

Reply:
For putting one line per option, I agree that it's certainly better for the diff.
I mean the extra empty lines added before and after variables, particularly those that are on 1 line (such as empty lines around `OPTFLAGS=-O2`): as far as I understand that should shouldn't matter for the diff. And if we start doing it everywhere consistently e.g. here https://github.com/iodide-project/pyodide/blob/6a4423356283a5d644c0e144f59570be0d1ba74c/Makefile#L5-L12 I find it a bit strange in terms of formatting.